src/index.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+
+const tasks = [
+  {
+    description: 'wash the Dishes',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'complete To Do list Project',
+    completed: false,
+    index: 2,
+  },
+];
+
+function renderTaskList() {
+  const taskList = document.getElementById('task-list');
+  taskList.innerHTML = '';
+  tasks.forEach((task) => {
+    const listItem = document.createElement('li');
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = task.completed;
+    checkbox.addEventListener('change', (event) => {
+      task.completed = event.target.checked;
+      // eslint-disable-next-line no-use-before-define
+      checkAllCompleted();
+    });
+    listItem.appendChild(checkbox);
+
+    const description = document.createElement('span');
+    description.innerText = task.description;
+    if (task.completed) {
+      description.classList.add('completed');
+    }
+    listItem.appendChild(description);
+
+    taskList.appendChild(listItem);
+  });
+}
```

Comment:
- [ ] Your function is populating the HTML with the data, but unfortunately it is not rendering by the order of the index of each item. Kindly refactor this function to make the list appear in order of the index values for each task as the project required.

Reply:
Hello I can't understand what you are saying can you explain with examples thankyou